lib/rubocop/formatter/json_formatter.rb

```diff
@@ -59,12 +59,15 @@ def hash_for_offense(offense)
       end
 
       # TODO: Consider better solution for Offense#real_column.
+      #       The minimum value of `start_column: real_column` is 1.
+      #       So, the minimum value of `last_column` should be 1.
+      #       And non-zero value of `last_column` should be used as is.
       def hash_for_location(offense)
         {
           start_line:   offense.line,
           start_column: offense.real_column,
           last_line:    offense.last_line,
-          last_column:  offense.last_column,
+          last_column:  offense.last_column.zero? ? 1 : offense.last_column,
```

Comment:
Sorry, I'm not familiar with the Rubocop codebase.

This change prevents a situation `start_column==1, last_column==0` from happening, but can something like `start_column==5, last_column==4` still happen?